hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastConfigBeanDefinitionParser.java

```diff
@@ -602,19 +605,51 @@ public void handleSymmetricEncryption(Node node, BeanDefinitionBuilder networkCo
         }
 
         public void handleExecutor(Node node) {
-            createAndFillListedBean(node, ExecutorConfig.class, "name", executorManagedMap);
+            BeanDefinitionBuilder execConfigBuilder = createBeanBuilder(ExecutorConfig.class);
+            fillAttributeValues(node, execConfigBuilder);
+            for (Node childNode : childElements(node)) {
+                String nodeName = cleanNodeName(childNode);
+                if ("quorum-ref".equals(nodeName)) {
+                    execConfigBuilder.addPropertyValue("quorumName", getTextContent(childNode));
+                }
+            }
+            executorManagedMap.put(getAttribute(node, "name"), execConfigBuilder.getBeanDefinition());
         }
 
         public void handleDurableExecutor(Node node) {
-            createAndFillListedBean(node, DurableExecutorConfig.class, "name", durableExecutorManagedMap);
+            BeanDefinitionBuilder execConfigBuilder = createBeanBuilder(DurableExecutorConfig.class);
+            fillAttributeValues(node, execConfigBuilder);
+            for (Node childNode : childElements(node)) {
+                String nodeName = cleanNodeName(childNode);
+                if ("quorum-ref".equals(nodeName)) {
+                    execConfigBuilder.addPropertyValue("quorumName", getTextContent(childNode));
+                }
+            }
+            durableExecutorManagedMap.put(getAttribute(node, "name"), execConfigBuilder.getBeanDefinition());
         }
 
         public void handleScheduledExecutor(Node node) {
-            createAndFillListedBean(node, ScheduledExecutorConfig.class, "name", scheduledExecutorManagedMap);
+            BeanDefinitionBuilder execConfigBuilder = createBeanBuilder(ScheduledExecutorConfig.class);
+            fillAttributeValues(node, execConfigBuilder);
+            for (Node childNode : childElements(node)) {
+                String nodeName = cleanNodeName(childNode);
+                if ("quorum-ref".equals(nodeName)) {
+                    execConfigBuilder.addPropertyValue("quorumName", getTextContent(childNode));
+                }
+            }
+            scheduledExecutorManagedMap.put(getAttribute(node, "name"), execConfigBuilder.getBeanDefinition());
         }
 
         public void handleCardinalityEstimator(Node node) {
-            createAndFillListedBean(node, CardinalityEstimatorConfig.class, "name", cardinalityEstimatorManagedMap);
+            BeanDefinitionBuilder execConfigBuilder = createBeanBuilder(CardinalityEstimatorConfig.class);
```

Comment:
The `createAndFillListedBean` also filled the `backup-count` and `async-backup-count` elements. Can you add those and add a test that verifies that parsing is still working as expected.

Reply:
That is true, that's a bug from my site. Fixed.